src/storage/mod.rs

```diff
@@ -919,6 +919,69 @@ impl Storage {
             .flatten()
     }
 
+    pub fn async_raw_batch_get(
+        &self,
+        ctx: Context,
+        keys: Vec<Key>,
```

Comment:
For all raw api, we use Vec<u8>.

Reply:
All fixed. Let's see what limit semantic we should use for raw_batch_scan if itself is actually reasonable. And I will combine all these as a new single commit.